src/web/reducers/flows.js

```diff
@@ -361,6 +377,87 @@ reducer = reduceReducers(
         flowsByName: _.omit(state.flowsByName, name)
       }),
 
+      // Inserting a new skill essentially:
+      // 1. creates a new flow
+      // 2. creates a new "skill" node
+      // 3. puts that new node in the "insert buffer", waiting for user to place it on the canvas
+      [insertNewSkill]: (state, { payload }) => {
+        const skillId = payload.skillId.replace(/^botpress-skill-/i, '')
+        const flowRandomId = nanoid(5)
```

Comment:
This looks like the 2nd or 3rd different way to generate random IDs :) Let's unify?

Reply:
Agreed, this one was introduced by Alex last week and I'd like to make the switch to this one as we go